evcouplings/align/protocol.py

```diff
@@ -1003,6 +1002,148 @@ def jackhmmer_search(**kwargs):
 
     return outcfg
 
+def hmmbuild_and_search(**kwargs):
+    """
+    Protocol:
+
+    Hmmbuild and hmmsearch against a sequence database.
```

Comment:
Add to documentation: If the user provides a pre-made HMM file, search the sequence database using that HMM. Otherwise, construct the HMM using the input alignment file, and then use the HMM to search the sequence database.